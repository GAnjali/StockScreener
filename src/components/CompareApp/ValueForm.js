import React from 'react';

const ValueForm = (props) => {
    return (
        <div>
            <h3 className={"input-label"}>{props.label}</h3>
            <input className="text-input" placeholder={props.placeholder} value={props.amountToInvest}
                   onChange={props.handleChange}/>
        </div>
    );
};

export default ValueForm;