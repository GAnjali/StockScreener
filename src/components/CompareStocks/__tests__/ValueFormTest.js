import {shallow} from "enzyme";
import React from "react";
import ValueForm from "../ValueForm";

describe("ValueFormTests", () => {
    it("Should render without crashing", () => {
        shallow(<ValueForm/>);
    });

    it('should match the ValueForm snapshot', function () {
        const valueFormComponent = shallow(<ValueForm/>);
        expect(valueFormComponent).toMatchSnapshot();
    });
});