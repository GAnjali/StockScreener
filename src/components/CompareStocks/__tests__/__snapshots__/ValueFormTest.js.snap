// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValueFormTests should match the InputField snapshot 1`] = `ShallowWrapper {}`;
